sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -32,6 +34,15 @@ export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResul
     sentiment: DocumentSentimentLabel;
 }
 
+// @public (undocumented)
```

Comment:
not familiar with this file, but previous entries don't have the `(undocumented` on it

Reply:
done.